test/blackbox-tests/test-cases/pkg/pin-stanza/pin-depopts.t

```diff
@@ -37,7 +37,5 @@ https://github.com/ocaml/dune/issues/10739
   > (lang dune 3.16)
```

Comment:
Remember to update the comment above this part of test claiming that this doesn't work correctly.